src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -63,7 +63,9 @@ public static void main(final String[] args) throws InterruptedException {
         final MantaMonitorServletModule mantaMonitorServletModule = new MantaMonitorServletModule();
         final Injector injector = Guice.createInjector(module);
         final Configuration configuration = injector.getInstance(Configuration.class);
-        final JettyServerBuilderModule jettyServerBuilderModule = new JettyServerBuilderModule(jettyServerPort);
+        final MantaClient client = injector.getInstance(MantaClient.class);
```

Comment:
An instance of `MantaClient` is created, but it is never close. Since the app is just exiting, this probably wouldn't lead to a resource leak, but as a matter of good practice, I would suggest that you close this resource when you finish using it.

Reply:
Additionally, it is a bit confusing regarding how this `MantaClient` instance will be used in comparison with how the instance returned from `MantaClientProvider` will be used.

Reply:
This ```MantaClient``` instance is still returned from the ```MantaClientProvider``` by the guice DI. So no new instances are created and ehe same instance of ```MantaClient``` is used to first get the keystore and truststore files and then execute the commands in the chain.

Reply:
Probably I can add a comment before the line#66 to clarify that the same ```MantaClient``` instance, as provided by the ```MantaClientProvider```, is used to get the JSON config file as well as the keystore and truststore files (in case of TLS) and also to execute commands in the chain.